### ISSUE TITLE
(parity-ui:1693): Gtk-WARNING **: cannot open display: #8971

### ISSUE THREAD

**Issue by stone212** [parity-js/shell#147]
_Before filing a new issue, please **provide the following information**._

> I'm running:
>
> - **Parity client version**: 1.10.x
> - **Which Parity version?**: Parity UI (also confusingly called Parity-JS): Parity UI v0.1.4-stable
> - **Which operating system?**: Debian
> - **How installed?**: via .deb
> - **Are you fully synchronized?**: yes
> - **Which network are you connected to?**: ethereum 
> - **Did you try to restart the node?**: yes

_Your issue description goes here below. Try to include **actual** vs. **expected behavior** and **steps to reproduce** the issue._

_Your issue description goes here below. Try to include **actual** vs. **expected behavior** and **steps to reproduce** the issue._

1. Install Parity-UI

2. type "sudo apt-get -f install" to install dependencies

3. type "sudo apt-get install libasound2" because that is not included somehow

4. type "parity-ui" as root or as parity user

Result:

`(parity-ui:1693): Gtk-WARNING **: cannot open display:`

Preferred Result:

Parity UI should say "Running" or something and I can then go to localhost:8108 and see the familiar Parity UI.

**Comment by Tbaut** [comment]
Can you try with the latest version https://github.com/parity-js/shell/releases ?

**Comment by stone212** [comment]
@Tbaut I am using the latest (stable) version.  Do you want me to try the beta?  This is a production server I am testing on which is why I chose stable.

**Comment by Tbaut** [comment]
Parity UI is independent of the node. Running Parity UI in beta should just affect yourself.
The localhost:8180 is deprecated in favor of Parity UI.

It looks like you have an unusual environment as you should not need to install dependancies. Can you please elaborate on your system, is the node running locally, what exactly do you want to achieve with Parity UI?

**Comment by stone212** [comment]
@Tbaut 

> Parity UI is independent of the node. 

I am really confused.  Is there a document on this?  I am used to using the Parity UI as a part of the node.  You run the node and then you go to 127.0.0.1:8180 and you see the UI.  I did this every day for more than a year.  Then one day I upgraded and I couldnt' do it anymore and I don't see any docs that instruct me how to move forward.

> It looks like you have an unusual environment as you should not need to install dependancies. 

I did need to.  I installed Parity UI on the system that is running Parity (my node) because that's where I should run it right?  That's where the UI used to be so I'm guessing that's where it is now. 

That server runs Ubuntu 16.04 (headless, no X server installed obviously as it is a server).

I *thought* that what I wanted to achieve is to be able to go to 127.0.0.1:8180 and see the UI.  But if it runs different now that's fine.  Just please *tell me* because things used to work one way and now they don't and I can't just guess what is going on in the devs head.  A doc would be very useful.

**Comment by Tbaut** [comment]
The confusion comes from the fact that you are very much used to something that has now changed. You should see parity as a node, with a CLI interface purely only. It exposes some APIs. This node should sit on your server. 

The UI is now totally separated. It connects to whatever node you want with WS. This application should sit on your local/client computer and connect to your server node through SSH tunneling as described here: https://wiki.parity.io/Wallet-Remote-Access.html

**Comment by stone212** [comment]
@Tbaut 

> The confusion comes from the fact that you are very much used to something that has now changed

Well actually the confusion comes from the fact that documentation on the change is hard to find.  I searched the wiki, etc.  It might help if what you just wrote was written somewhere else like on the page that says I need to download the UI.  Or somewhere I can search for within the Wiki (the link you sent did not come up when I searched) or on the Parity UI wiki page of all places.

But I consider this issue closed also.  Thank you for your reply.  (I do have another issue that I'll post as another Issue).
